daemon/devices.go

```diff
@@ -30,6 +30,12 @@ func (daemon *Daemon) handleDevice(req container.DeviceRequest, spec *specs.Spec
 			}
 		}
 	} else if dd := deviceDrivers[req.Driver]; dd != nil {
+		// We add a special case for the CDI driver here since this does do not require specific capabilities.
```

Comment:
"this does do not require" ? :eyes:

Reply:
I have reworded the comment to hopefully be clearer. Please let me know if more context should be added.